viz/PluginLoader.cpp

```diff
@@ -39,6 +41,8 @@ namespace vizkit3d {
 	    pluginNames->push_back("SonarVisualization");
 	    pluginNames->push_back("DepthMapVisualization");
 	    pluginNames->push_back("DistanceImageVisualization");
+		pluginNames->push_back("RigidBodyStateSE3Visualization");
+		pluginNames->push_back("WrenchVisualization");
```

Comment:
Wrong indentation (happens elswhere in the patch).

Rock style guide is 4 space indent, no tabs. I'm guessing that you are using tabs.